src/resources/views/inc/form_save_buttons.blade.php

```diff
@@ -23,4 +23,88 @@
     </div>
 
     <a href="{{ url($crud->route) }}" class="btn btn-default"><span class="fa fa-ban"></span> &nbsp;{{ trans('backpack::crud.cancel') }}</a>
-</div>
\ No newline at end of file
+</div>
+
+@push('crud_fields_scripts')
+<script>
+    jQuery('document').ready(function($){
+
+      // Save button has multiple actions: save and exit, save and edit, save and new
+      var saveActions = $('#saveActions'),
+      crudForm        = saveActions.parents('form'),
+      saveActionField = $('[name="save_action"]');
+
+      saveActions.on('click', '.dropdown-menu a', function(){
+          var saveAction = $(this).data('value');
+          saveActionField.val( saveAction );
+          crudForm.submit();
+      });
+
+      // Ctrl+S and Cmd+S trigger Save button click
+      $(document).keydown(function(e) {
+          if ((e.which == '115' || e.which == '83' ) && (e.ctrlKey || e.metaKey))
+          {
+              e.preventDefault();
+              // alert("Ctrl-s pressed");
```

Comment:
idk, i don't change this https://github.com/Laravel-Backpack/CRUD/blob/master/src/resources/views/form_content.blade.php#L50